pom.xml

```diff
@@ -778,7 +778,7 @@
                     <plugin>
                         <groupId>org.owasp</groupId>
                         <artifactId>dependency-check-maven</artifactId>
-                        <version>1.2.9</version>
+                        <version>1.4.5</version>
```

Comment:
Nice, did you look at any new configuration settings? https://jeremylong.github.io/DependencyCheck/dependency-check-maven/configuration.html